gateway-developer-plugin/src/test/resources/multi-project-with-assertions-dependencies/build.gradle

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+plugins {
+    id 'com.ca.apim.gateway.gateway-developer-plugin' apply false
+}
+
+subprojects {
+    apply plugin: 'com.ca.apim.gateway.gateway-developer-plugin'
+
+    group 'com.ca'
+    version '1.2.3-SNAPSHOT'
+
+    repositories {
+        flatDir {
+            dirs "../lib"
+        }
+        maven {
+            url "file:///" + project.property('jarDir')
```

Comment:
I may be mistaken. Is there supposed to be 3 backslashes?

Reply:
I guess yes, @J-Lou, due to this:
```"-PjarDir=" + System.getProperty("user.dir") + "/build/test-mvn-repo"```

The System.getProperty might return the user.dir without the starting backslash